### DISCUSSION TITLE
Difference between User Roles and how to change them?

### DISCUSSION THREAD

**Discussion by SteffoSpieler** [tolgee/tolgee-platform#1515] [Q&A] [answered]
First up - I'm running tolgee via docker compose on my linux server.
I've created my account on the registration page. (I had enabled registration for a short time to let me create my account.)

I've read in #1263 that there are some user roles. What do they do?
Also there it says `if you modify the in the DB`, how would I do that?

**Comment by JanCizmar** [comment, marked as answer]
Hey, there is the ADMIN role and USER role. 

**ADMIN** user can access the server administration:
- See all user, debug their accounts, remove them
- See all organizations and their projects

**USER** is an ordinary user. Their permissions are set in the platform on the organization / project level.
